.github/workflows/icu_exhaustive_tests.yml

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Actions configuration for Exhaustive Tests for ICU.
+# 
+# Note: The exhaustive test configuration is in a separate file
+# so that it can be run independently from the regular builds.
+#
+# To run these tests, comment on a PR with "run exhaustive tests" 
+# or manually trigger the workflow. If triggered from a PR comment,
+# it adds checks to the PR and updates them with the results.
+
+name: Exhaustive Tests for ICU
+
+on:
+  workflow_dispatch:
+  issue_comment:
+    types: [created]
```

Comment:
The limitation with `issue_comment ` is that it will trigger a lot of skipped tests (on each comment on a PR).
We discussed this and agreed that running the tests manually is possibly the most simple solution.
I have updated the docs accordingly (Thanks for pointing those out).